webservices/rest.py

```diff
@@ -52,13 +52,8 @@ def sqla_conn_string():
 def limit_remote_addr():
     falses = (False, 'False', 'false', 'f')
     if FEC_API_WHITELIST_IPS not in falses :
-        remote = request.remote_addr
-        route = list(request.access_route)
-        while remote in trusted_proxies:
-            remote = route.pop()
-        print(route.pop())
-
-        if remote not in trusted_proxies:
+        route = request.access_route
+        if route[-2] not in trusted_proxies:
```

Comment:
Is this because `route[-1]` is the CF router, `route[-2]` is the api.data.gov IP, and `route[:-2]` is stuff we don't care about? If that's right, it would be helpful to clarify what's going on, and possibly handle the case where `access_route` is too short:

```python
try:
    *_, api_data_route, cf_route = request.access_route
except ValueError:  # Not enough routes
    abort(403)
else:
    if api_data_route not in trusted_proxies:
       abort(403)
```